electrum.php

```diff
@@ -105,6 +109,88 @@ public function createnewaddress() : string {
 		return $response;
 	}
 
+	/**
+	 * Gives a JSON array of all current addresses.
+	 * 
+	 * @return array List of addresses
+	 */
+	public function listaddresses() : array {
+		$response = $this->curl("listaddresses", []);
+
+		return $response;
+	}
+
+	/**
+	 * Gives a specified address balance.
+	 * 
+	 * @param string $address to get balance
+	 * @param bool $confirmed_only only show confirmed BTC
+	 * 
+	 * @return float Address balance
+	 */
+	public function getaddressbalance(string $address, bool $confirmed_only = false) : float {
+		$response = $this->curl("getaddressbalance", ["address"=>$address]);
+		
+		$total = 0.0;
+
+		if (!$confirmed_only && key_exists("unconfirmed", $response)) $total += $response["unconfirmed"];
+		if (key_exists("confirmed", $response)) $total += $response["confirmed"];
+
+		return $total;
+	}
+
+	/**
+	 * Gives the history of a specified address.
+	 * 
+	 * @param string $address to get the history
+	 * 
+	 * @return array List of address history
+	 */
+	public function getaddresshistory(string $address) : array {
+		$response = $this->curl("getaddresshistory", ["address"=>$address]);
+
+		return $response;
+	}
+
+	/**
+	 * Freezes a specified address from recieving or sending BTC.rotating
```

Comment:
> or sending **BTC.rotating**

This does not look right, also you can't really freeze an address from receiving ;-)

```suggestion
	 * Freezes a specified address from sending BTC.
```